### ISSUE TITLE
Support more keyboard operations

### ISSUE THREAD

**Issue by seawind543** [seawind543/react-token-input#57]
Support more keyboard operations, such as **ArrowLeft** to select a token and `Enter` to make the token enter inline-editing mode

**Comment by isaachinman** [comment]
Hi @seawind543 – it would also be nice to be able to use the arrow keys to navigate between tags. Let's assume I have added three tokens, but then want to delete the middle one. If I understand correctly, the only way to do that at the moment is with the mouse, right?

**Comment by seawind543** [comment]
@isaachinman Yes, you are right! Currently, we do not support the `selected` state on the token either.

**Comment by isaachinman** [comment]
Thanks. I ended up writing my own component, as keyboard navigation was a requirement.
